esm/interface/custom-element-registry.js

```diff
@@ -195,7 +195,7 @@ export class CustomElementRegistry {
    * @param {Function} Class **Class** of custom element
    * @returns {string?} found tag name or null
    */
-    getName(Class){
+    getName(Class) {
```

Comment:
this is also good but I wasn't clear ... see indentation compared to get(...)

![Uploading Screenshot from 2023-12-12 13-33-21.png…]()

Reply:
O.k. -- I finally got it! :)

Indentation should be fixed now.